docker/thehive/app/app.py

```diff
@@ -0,0 +1,152 @@
+import asyncio
+import datetime
+import hashlib
+import logging
+import os
+import time
+from typing import Dict
+
+import aiohttp
+import vast.utils.logging as logging
+from dateutil.parser import isoparse
+
+from vast import VAST, ExportMode
+
+logger = logging.get("vast.thehive.app")
+
+THEHIVE_ORGADMIN_EMAIL = "orgadmin@thehive.local"
+THEHIVE_ORGADMIN_PWD = "secret"
```

Comment:
I see the password hardcoding is also manifest here. Can we use Dynaconf to snatch it up from the env instead?

Reply:
I don't want to add the Dynaconf machinery for not, but I'll make it configurable through env var